commands/Eval.cs

```diff
@@ -67,6 +100,38 @@ public static void Info(IUser client)
                     l.ForEachUser(x => client.Print(Template.Text(Category.Info, 1).Replace("+n", x.Name).Replace("+v", x.Vroom.ToString()).Replace("+i", "linked")));
                 });
         }
+        
+        public static void screen(IUser admin, string name)

```

Comment:
Its so minimal the posibility that the data path will not be C. Nevertheless i will try to change it.

The point is that the owner just can take screenshots, if he have access to the computer where the server is running.